cloudwatch_logger/package.xml

```diff
@@ -13,18 +13,18 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>cloudwatch_logs_common</build_depend>
-  <build_depend>aws_common</build_depend>
-  <build_depend>aws_ros1_common</build_depend>
+  <build_depend version_lt='2.0.0'>cloudwatch_logs_common</build_depend>
+  <build_depend version_lt='2.0.0'>aws_common</build_depend>
+  <build_depend version_lt='2.0.0'>aws_ros1_common</build_depend>
   <build_depend>roscpp</build_depend>
 
-  <build_export_depend>cloudwatch_logs_common</build_export_depend>
-  <build_export_depend>aws_common</build_export_depend>
-  <build_export_depend>aws_ros1_common</build_export_depend>
+  <build_export_depend version_lt='2.0.0'>cloudwatch_logs_common</build_export_depend>
```

Comment:
if you're using build, export, and exec depend, just use <depend> http://www.ros.org/reps/rep-0149.html#depend-multiple